### ISSUE TITLE
bug(sort): Screenreaders not reading out current sort state

### ISSUE THREAD

**Issue by andrewseguin** [angular/components#20077]
Basic sort example: https://stackblitz.com/angular/qmbrjdbmgdo?file=src%2Fapp%2Fsort-overview-example.ts

Using Chromevox, focus on a column header. Screenreader reports: "Fat (g) Button row 1 column 3 Fat (g) Press Search plus Space to activate)"

No mention of the current sort state. Was this a regression for the merged fix https://github.com/angular/components/pull/13297? Even when navigating across data cells I cannot get it to tell me how its sorted

**Comment by andrewseguin** [comment]
Unfortunately this is an issue with ChromeVox - https://bugs.chromium.org/p/chromium/issues/detail?id=987997

Also VoiceOver does not announce the `aria-sort` yet, either.

This does work with NVDA and appears to be supported in JAWS. Our HTML is correct, but the screenreaders need to catch up to the standards. 

A workaround is to add labels with `aria-label` to the headers, but this may result in a double-announce for screenreaders that are supporting `aria-sort`.
